modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,99 +82,562 @@ The following charts are available only if you are using Unified Agent and have
+The average response time of Object Store connector requests, grouped by operation and automatic intervals.
+
+Object Store Response Time::
+The average number of Object Store connector request failures, grouped by operation and automatic intervals.
+
+Object Store Failures::
+The average number of Anypoint MQ connector inbound requests, grouped by operation and automatic intervals.
+
+Anypoint MQ Operations - Inbound::
+The average response time of Anypoint MQ connector inbound requests, grouped by operation and automatic intervals.
+
+Anypoint MQ Response Time - Inbound::
+The average number of Anypoint MQ connector inbound request failures, grouped by operation and automatic intervals.
+
+Anypoint MQ Failures - Inbound::
+The average number of Anypoint MQ connector outbound requests, grouped by operation and automatic intervals.
+
+Anypoint MQ Operations - Outbound::
+The average response time of Anypoint MQ connector outbound requests, grouped by operation and automatic intervals.
+
+Anypoint MQ Response Time - Outbound::
+The average number of Anypoint MQ connector outbound request failures, grouped by operation and automatic intervals.
+
+Anypoint MQ Failures - Outbound::
+The average number of requests of connectors of type other, grouped by operation in automatic intervals.
+
+Other Connector Requests::
+The average number of requests of connectors of type other, grouped in automatic intervals.
+
+Other Connector Response Time::
+The average response time of requests for connectors of type other, grouped in automatic intervals.
+
+Other Connector Failures::
+The average number of requests failures of connectors of type other, grouped in automatic intervals.
+
+[[performance-charts]]
+== Performance
+
+The following charts appear on the *Performance* dashboard:
+
+Inbound - Average Response Time::
+The average response time of inbound requests, grouped in automatic intervals.
+
+Inbound - Average Response Time Grouped by Endpoint::
+The average, maximum, and minimum response times, grouped by endpoint. Cells will vary in color according to the following thresholds:
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Mule Messages::
+The number of Mule Messages processed in the runtime, grouped in automatic intervals.
+
+Inbound Response Time graphs (99, 90, 75, and 50 Percentile)::
+The response time of the 99th, 90th, 75th, and 50th percentile of inbound requests, grouped by application in automatic intervals.
```

Comment:
Notice that presentation changes here; please standardize percentile language GLOBALLY.